src/app/inventory/store/sockets.ts

```diff
@@ -236,17 +236,20 @@ function buildDefinedSocket(
         const plugs: {
           [plugItemHash: number]: DestinyItemSocketEntryPlugItemRandomizedDefinition;
         } = {};
-        for (const reusablePlug of plugSet.reusablePlugItems) {
-          const existing = plugs[reusablePlug.plugItemHash];
-          if (!existing || (!existing.currentlyCanRoll && reusablePlug.currentlyCanRoll)) {
-            plugs[reusablePlug.plugItemHash] = reusablePlug;
+        for (const randomPlug of plugSet.reusablePlugItems) {
```

Comment:
This rename makes me nervous because I'd think `randomPlug` would be plugs from the `randomizedPlugSetHash`

Reply:
it is.

Reply:
see line 232

Reply:
Ah I see. The plugSet definitions don't help here.